CHANGES/5367.feature

```diff
@@ -0,0 +1 @@
+Add a retention policy feature - when specified, the latest N versions of each package will be kept and older versions will be purged.
```

Comment:
I mentioned elsewhere but, "retention policy" still seems like an apt description of the feature in noun form, so I'm not sure if we want to adjust this wording as well?

Reply:
This prompted me to think about how users will learn about new configuration.
What about adding to the docs what is the parameter and how to use it?

Reply:
Done